relayer/src/supervisor.rs

```diff
@@ -273,6 +273,11 @@ impl Supervisor {
             }
         }
 
+        // At least one chain runtime should be alive and kicking, otherwise bail.
```

Comment:
funny comment but what does it mean exactly?

Reply:
I meant to say that if the vector of chain runtimes (`self.registry`) is empty, that means no chain runtime could be instantiated, so the supervisor (i) cannot spawn any worker and (ii) has no subscription to any chain to `select` on. I'll use more accurate language here!

Reply:
also https://github.com/informalsystems/ibc-rs/issues/972#issuecomment-847185935
I am not able to see that panic. What are the steps to see that? I tried to start hermes without any chains started and also tried to kill the chain procs while hermes is running.